drivers/hmis/app/graphql/types/hmis_schema/submit_form_result.rb

```diff
@@ -23,9 +23,9 @@ class HmisSchema::SubmitFormResult < Types::BaseUnion
 
     def self.resolve_type(object, _context)
       config = Hmis::Form::Definition::FORM_ROLE_CONFIG.find do |_, value|
-        value[:class_name] == object.class.name
+        value[:class_name] == object.class.sti_name
```

Comment:
Hmm, I'm not sure what `value[:class_name]` will be....

Reply:
It will be a constant like Hmis::Hud::Client, so I guess this one should be `name`.